---
title:  "Paper: Cell lineage-specific mitochondrial resilience during mammalian organogenesis"
categories:
  - Blog
tags:
  - research
  - publication
  - mtDNA
---

The most collaborative project I have contributed to so far was recently [published](https://www.sciencedirect.com/science/article/pii/S0092867423000934?via%3Dihub), to the joy of pretty much everyone I work with. It was a large team effort, with Stephen Burr and Angelos Glynos leading on the lab side, and [Malwina Prater](https://github.com/nmalwinka) and [Florian Klimm](https://floklimm.github.io/index.html) in charge of bioinformatics and data analysis. My role in the project was tiny: I had been independently curious about some of the data coming out from our mouse facility, which then fed into this multifaceted interdisciplinary piece of work.

The paper is ostensibly about mitochondrial function and early embryonic development. For a long time we have known that mitochondrial activity can differ significantly between organs. By analysing single-cell data from mouse embryos, my colleagues show that this difference arises very early in development, before organ formation. They also show that pathogenic mtDNA mutations already induce compensatory responses at that early stage, and that those responses are both tissue and mutation-specific. These compensatory mechanisms may go some way towards explaining why mitochondrial diseases affect tissues so differently, and consequently may also elucidate potential targets for organ-directed treatments. 

In addition to laying out the case for tissue-specificity in mitochondrial function early in development, the paper also introduces a new animal model for mitochondrial disease. I have previously written about mice carrying the heteroplasmic m.5024C>T mutation on the acceptor stem of the mitochondrial tRNA-Ala ([Kauppila _et al._, 2016](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5039181/ )). A lot of animal research into mitochondrial function and the effects of pathogenic mutations is carried out with m.5024C>T mice, and this is also one of the mutations investigated in our new paper. In addition, we further introduce a new m.5019A>G mouse model, which has another heteroplasmic mutation on the same gene. While the two mutations are barely five base pairs apart, they present differently in a number of important ways. For example, higher heteroplasmy fractions of the m.5019A>G mutation are tolerated compared to m.5024C>T. 

Why would we want more animal models? One of my main research interests in the lab is understanding heteroplasmy dynamics, i.e. when, how, and why the fraction of mutant mtDNA in a sample (or cell, or organism) changes. The mechanisms driving these dynamics may be general, present across many or all mtDNA mutations, or they may be specific to particular mutations and their functional effects. Studying multiple related mouse models enables us to differentiate between the general and the specific. I suspect going forward we will see a lot more comparative work being done between m.5024C>T and m.5019A>G, both in our lab, but also more widely in the field.